### ISSUE TITLE
Hosting bundle installer link is pulling down an old installer

### ISSUE THREAD

**Issue by guardrex** [aspnet/Docs#5688]
@joeloff I thought the new link we're using for the hosting bundle installer would auto roll-forward to the latest hosting bundle installer. However, it's still pulling down the one with 2.0.5 (I assume there's a new one because the 2.0.6 runtime is out).

https://aka.ms/dotnetcore-2-windowshosting

... downloads `DotNetCore.2.0.5-WindowsHosting.exe`.

Also, I see they revamped the .NET Downloads page and the hosting bundle installer isn't available there any longer. https://www.microsoft.com/net/download/all Do you know if it's available anywhere outside of the docs now?

cc/ @shirhatti

**Comment by shirhatti** [comment]
@guardrex I've created an issue on the dot.net website to re-add the link. I have no idea when it was dropped

**Comment by guardrex** [comment]
It probably wasn't too long ago. I think I looked at the downloads page just within the past 2-3 weeks.

btw-- Is that dot.net website on a GH repo where there are issues? I have an issue I'd like to open with them that pertains to some of the .NET site pages that lead to this doc set.

**Comment by shirhatti** [comment]
Ahh! It is available. It's just a little hard to discover after the redesign
https://www.microsoft.com/net/download/all -> https://www.microsoft.com/net/download/dotnet-core/runtime-2.0.6 -> https://www.microsoft.com/net/download/thank-you/dotnet-runtime-2.0.6-windows-server-hosting-installer

**Comment by masterjs** [comment]
Ok! Thank you for the information, its very hard to discover compared to before.. I'm hoping there's going to be someone that will eventually make this pretty again ;o)

**Comment by shirhatti** [comment]
@guardrex To answer your earlier question, see https://github.com/aspnet/Announcements/issues/295#issuecomment-373117653
Tl;dr 2.0.6 ServerHosting  bundle is currently being rebuilt

**Comment by guardrex** [comment]
ty @shirhatti ... I'll keep an :eye: out and get the links in the topics updated as soon as we have it.

Still tho, I was mistaken (apparently) about `https://aka.ms/dotnetcore-2-windowshosting` ... I thought we had a good roll-forward link there for all 2.x hosting bundle releases such that I wouldn't need to update the docs every time there's a new release.

**Comment by AlejandroFlorin** [comment]
Tried to install the 2.0.6 hosting bundle from the links above and I get an error with the following log:

**Comment by guardrex** [comment]
On stand-by for the new link. I'll get it posted as soon as it's available.

https://github.com/aspnet/IISIntegration/issues/658#issuecomment-372764851
